projects/palindrome/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Palindrome in Every Language
+layout: default
+date: 2020-10-07
+last-modified: 2020-10-07
+featured-image: 
+tags: [panlindrome]
+authors:
+  - anohene1
+---
+
+This is an article on how to check if a number is a palindrome in every language.
+
+## Description
+
+A palindrome is a number that reads the same backward and forward.
```

Comment:
I think we should open this up to any string. What do you think? I realize that may mess with your current solution which leverages mod.

Reply:
The other issue here is the limit in size. Numbers have a limit in most languages.

Reply:
I will change the Int variables to Long to fix the issue of the limit in size.

Reply:
I will try and open it up to Strings sometime in the future.

Reply:
I have finished changing all the Int variables to Long.

Reply:
Okay! Maybe we can rename the project to Palindromic Number to be specific.

Reply:
Sure. I'll go ahead and do that